packages/chrome/src/elements/sheet/Sheet.spec.tsx

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useRef, useState, forwardRef } from 'react';
+import userEvent from '@testing-library/user-event';
+import { render } from 'garden-test-utils';
+import { ThemeProvider, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+import { Sheet, ISheetProps } from './Sheet';
+
+describe('Sheet', () => {
+  const Example = forwardRef<HTMLElement, ISheetProps>((props: ISheetProps, ref) => {
+    const [isOpen, setIsOpen] = useState(() => props.isOpen);
```

Comment:
Could you clarify the purpose of using lazy initial state?

Reply:
Good call on this, it isn't necessary. Will update